coremltools/converters/mil/frontend/torch/utils.py

```diff
@@ -16,21 +14,41 @@ class TorchFrontend(Enum):
 def sanitize_op_kind(op_kind: str) -> str:
     """
     In our torch converter, we register torch ops only by its "canonical" name:
-    1. No namespace prefix if it is the common aten/prim: e.g. ``aten::softmax`` -> ``softmax``
-    2. No double underscore prefix and suffix: e.g. ``__add__`` -> ``add``
-    3. Lower-case characters only: e.g. ``mul.Tensor`` -> ``mul.tensor``
+    1. Lower-case characters only, e.g. ``div.Tensor`` -> ``div.tensor``
+    2. No double underscore prefix and suffix, e.g. ``__add__`` -> ``add``
+    3. No namespace prefix if it is the common aten/prim, e.g. 
+           ``aten::softmax`` -> ``softmax``
+            ``aten.pow`` -> ``pow``
+       and no type trait suffix if it is not distinguished in CoreML, e.g.
+           ``bmm.default`` -> ``bmm``
+           ``slice_copy.tensor`` -> ``slice_copy``
+           ``mul.scalar`` -> ``mul``
     """
-    # 1. Skip the aten/prim namespace prefix
-    namespace = op_kind.split("::")[0].lower()
-    if namespace in _DEFAULT_OP_NAMESPACES:
-        op_kind = op_kind.split("::")[-1]
+    # 1. Lower case
+    op_kind = op_kind.lower()
 
     # 2. Remove underscore prefix and suffix
     if op_kind.startswith("__") and op_kind.endswith("__"):
         op_kind = op_kind[2:-2]
 
-    # 3. Lower case
-    return op_kind.lower()
+    # 3. Skip the aten/prim namespace prefix, and default/tensor/scalar suffix
+    def skip_default_prefix_and_suffix_with_deliminator(op_kind: str, deliminator: str) -> str:
```

Comment:
The inner function should be put in the very beginning of a function

Reply:
Revised as suggested